File: API/Cinema.DB/Models/Movie.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Cinema.DB.Models
{
    public class Movie
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public int DurationHours { get; set; }
        public int DurationMinutes { get; set; }
    }
}

File: Angular/cinema/src/models/Movie.ts
export class Movie {
    id: string;
    title: string;
    durationHours: number;
    durationMinutes: number;

    constructor() {
        this.id = '',
        this.title = '',
        this.durationHours = 0,
        this.durationMinutes = 0
    }
}